SECURITY.md

```diff
@@ -2,19 +2,13 @@
 
 ## Supported Versions
 
-| Version     | Supported          | Status                | Comments                             |
-| ----------- | ------------------ | --------------------- | ------------------------------------ |
-| 1.8.x       | :white_check_mark: | Long Term Support     | See [Long Term Support policy][0]    |
-| 1.3.x-1.7.x | :x:                |                       |                                      |
-| 1.2.x       | :warning:          | Security patches only | Last version to provide IE 8 support |
-| <1.2.0      | :x:                |                       |                                      |
-
-## Reporting a Vulnerability
+[Support for legacy AngularJS has fully ended.](https://goo.gle/angularjs-end-of-life).
 
-Email us at [security@angularjs.org](mailto:security@angularjs.org) to report any potential security issues in AngularJS.
+[Visit https://angular.io for the actively supported Angular framework](https://angular.io).
 
-Please [use the latest AngularJS possible](https://docs.angularjs.org/guide/security#use-the-latest-angularjs-possible)
-and keep in mind the guidance around AngularJS'
-[expression language](https://docs.angularjs.org/guide/security#angularjs-templates-and-expressions).
-
-[0]: https://docs.angularjs.org/misc/version-support-status#long-term-support
+| Version     | Supported          | Status                | Comments                             |
+| ----------- | ------------------ | --------------------- | ------------------------------------ |
+| 1.8.x       | :x:                | All support ended     |                                      |
```

Comment:
Add dates in the comments so that people have a little bit more context.

Reply:
I've added "as of January 2022" in the paragraph text above